docs/nym-simple-websocket-tutorial.md

```diff
@@ -0,0 +1,1190 @@
+                    <div class="field" style="margin-top: 10px;">
+                        <button class="ui button" id="send-button"><i class="icon location arrow"></i>Send</button>
+                    </div>
+                </div>
+                
+                <div class="ui icon message" style="margin-left:20px;max-width: fit-content;">
+                    <i class="question circle icon"></i>
+                    <div class="content">
+                        <div class="header">
+                            How it works
+                            </div>
+                            <p>Once you have started your Nym Websocket client(s), you can fill out the form and send data to the mixnet using the <b>"Send"</b> button.</p>
+                            <p>Your message will then be relayed through your Nym Websocket client running on the port (specified using --port in the command line) which is set to 1977 by default.</p>
+                            <p>Below, you can see the activity log. <b style='color: #36d481;'>Sent</b> messages will display in <b style='color: #36d481;'>green</b> while <b style='color: orange;'>received</b> messages will display in <b style='color: orange;'>orange</b>.</p>
+                    </div>
+                </div>
+                
+                <h3 style="margin-left:10px">Activity Log</h3>
+                
+                <p style="background-color: #202124;color: #fff;">
+                    <span id="output"></div>
+                </p>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.1/jquery.min.js" integrity="sha512-aVKKRRi/Q/YV+4mjoKBsE4x3H+BkegoM/em46NNlCqNTmUYADjBbeNefNxYV7giUp0VxICtqdrbqU7iVaeZNXA==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+                <script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.5.0/semantic.min.js" integrity="sha512-Xo0Jh8MsOn72LGV8kU5LsclG7SUzJsWGhXbWcYs2MAmChkQzwiW/yTQwdJ8w6UA9C6EVG18GHb/TrYpYCjyAQw==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+                <script src="index.ts"></script>
+            </body>
+        </html>
+
+    </details>
+
+    Save the file in the editor and lets go back to our console window. Lets go ahead and type :
+
+    ```
+    npm start
+    ```
+    Then, lets go back to [localhost:1234](http://localhost:1234/) and check the result. We should now have a new UI for our application!
+
+### Connecting our Nym Websocket Client
+
+At this point, we should have all our code at this point be able to make the first simple websocket connection that we are looking for. 
+Lets go ahead and get ourselves a copy of the Nym Websocket Client (the orange boxes that we see in our initial [diagram](#what-are-we-building), to the right of the blue 'User Client' box).
+
+
+Go to [Nym's releases page](https://github.com/nymtech/nym/releases), find the latest Binaries release , expand the ▶ Assets section and click on the ❒ `nym-client` option.
```

Comment:
Add instruction to either go to releases if there are binaries supported (ubuntu20 and mac iirc) or build it yourself and link to build instructions